doc/md_src_files/060_decision-trees.md

```diff
@@ -113,9 +113,30 @@ support crippled** as a default. *Exploitation* needs no special default; if ade
+Vulnerability management is a part of asset management.
+SSVC can benefit from asset management practices and systems, particularly in regard to automating data collection and answers for some decision points.
+SSVC depends on asset management to some extent, particularly for context on the cost and risk associated with changing or updating the asset.
+
+Asset management can help automate the collection of the [*Mission Impact*](#mission-impact), [*Situated Safety Impact](#situated-safety-impact), and [*System Exposure*](#system-exposure) decision points.
+These decision points tend to apply per asset rather than per vulnerability.
+Therefore, once each is assessed for each asset, it can be applied to each vulnerability that applies to that asset.
+While the asset assessment should be reviewed occasionally for accuracy, storing this data in an asset management system should enable automated scoring of new vulnerabilities on these decision points for those assets. 
+
+Our method is for prioritizing vulnerabilities based on the risk stemming from exploitation.
+There are other reasonable asset management considerations that may influence remediation timelines.
+There are at least three aspects of asset management that may be important but are out of scope for SSVC.
+First and most obvious is the transaction cost of conducting the mitigation or remediation.
+System administrators are paid to develop or apply any remediations or mitigations, and there may be other transactional costs such as downtime for updates.
+Second is the risk of the remediation or mitigation introducing a new error or vulnerability.
+Regression testing is part of managing this type of risk. Finally, there may be an operational cost of applying a remediation or mitigation, representing an ongoing change of functionality or increased overhead.
+A decision maker could order work within one SSVC priority class (scheduled, out-of-cycle, etc.) based on these asset management considerations, for example.
+Once the organization remediates or mitigates all the high-priority vulnerabilities, they can then focus on the medium-level vulnerabilities with the same effort spent on the high-priority ones.
+
+Asset management and risk management also drive some of the up-front work an organization would need to do to gather some of the necessary information.
+This situation is not new; an asset owner cannot prioritize which fixes to deploy to its assets if it does not know what assets it owns and their locations.
```

Comment:
I think the "and their locations" is not actually true. I can prioritize fixes to cloud servers having no knowledge of their location, just the fact that they're my responsibility is sufficient, no?

Reply:
Is calling a virtual address or identifier a location too much of a stretch for "location"?

Reply:
Identity is not location. If what is meant is identity, then it should say that.

The enumeration of what assets are deployed and which assets have vulnerable components is I think the thing we're getting at here. And that's a mapping of (asset-id has vul-id), or more likely it's mediated through (asset-id has component-id has vul-id).

Yes, location can be a kind of identity (e.g., IP address X has component version Y with vul Z is an adequate way of applying asset management to this problem). It's just not the only kind.

Reply:
How's https://github.com/CERTCC/SSVC/pull/114/commits/bc244f9d96aa31a80780a3cd21adb6f6e1778dc1 ?

Reply:
Yeah that works.